docker/caffe2/jenkins/common/install_ccache.sh

```diff
@@ -2,19 +2,6 @@
 
 set -ex
 
-# Install ccache from source.
-# Needs specific branch to work with nvcc (ccache/ccache#145)
-# Also pulls in a commit that disables documentation generation,
-# as this requires asciidoc to be installed (which pulls in a LOT of deps).
-pushd /tmp
-git clone https://github.com/pietern/ccache -b ccbin
-pushd ccache
-./autogen.sh
-./configure --prefix=/usr/local
-make "-j$(nproc)" install
-popd
-popd
```

Comment:
What's going on here?

Reply:
There are compilation errors when building ccache from source with devtoolset7 (maybe the ccache snapshot we are building is too old). Since currently we are using sccache in all CI jobs, skip installing ccache seems to be fine.